tests/prefetch_related/models.py

```diff
@@ -182,6 +182,15 @@ class Article(models.Model):
     name = models.CharField(max_length=20)
 
 
+class Animal(models.Model):
+    name = models.CharField(max_length=20)
+    weight = models.FloatField()
```

Comment:
```suggestion
```

Reply:
> 
@felixxm  We are using this property to test that if we don't pass evaluate this on the custom queryset we pass on prefetch of generic foreign key. Then we should be making a query to database.

Reply:
Ahh, yes :+1: